### DISCUSSION TITLE
Cannot sign Microsoft CLA before contributing

### DISCUSSION THREAD

**Discussion by faburaya** [microsoft/vcpkg#20741] [Q&A]
From latest versions of Mozilla Firefox (v59) or Google Chromium (v65) in Ubuntu 16.04 x64, the page for signing Microsoft CLA before contribution does not work.
The "Sign in" button does obligates me to log into my Github account, but then just reloads the page and nothing else happens.

**Comment by faburaya** [comment]
Also does not work from Edge or Opera browsers in Windows 10.

**Comment by kmhofmann** [comment]
Same here. I was puzzled by the 'Sign In' button as it didn't seem to do anything.

As a workaround, I was able to sign the agreement via the link provided by the PR bot.

**Comment by PhoebeHui** [comment]
@faburaya, does the issue still occur?

**Comment by faburaya** [comment]
Yes, @PhoebeHui 
It still occurs!

**Comment by takev** [comment]
The CONTRIBUTING.md page points to https://cla.opensource.microsoft.com which at least on Safari does not show any way of actually signing the CLA.

Thanks to @kmhofmann 's comment on this issue, I just went to a random pull request and got to a working CLA sign page through the PR-bot.

**Comment by PhoebeHui** [comment]
I think we need to update the docs https://github.com/microsoft/vcpkg/blob/master/CONTRIBUTING.md#legal, and give a guildline about signing the CLA.
